pkgs/top-level/python-packages.nix

```diff
@@ -29806,6 +29806,23 @@ EOF
     '';
   };
 
+  neovim-remote = buildPythonPackage rec {
```

Comment:
You are correct - I'm still getting used to the nixpkgs layout, and naively assumed that python-packages should contain all the packages written in Python.

I'll move it to a more appropriate location.